sdk/data/azcosmos/example_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
```

Comment:
```suggestion
package azcosmos_test
```

Reply:
You will need to import ""github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos" and prefix all SDK apis with "azcosmos.{API}", for example "azcosmos.NewCosmosClient"